packages/template/contentlayer.config.js

```diff
@@ -33,6 +33,10 @@ const computedFields = {
     /* eslint no-underscore-dangle: off */
     resolve: (post) => post._raw.flattenedPath,
   },
+  slug: {
+    type: "string",
+    resolve: (post) => post._raw.flattenedPath.replace(/^.+?(\/)/, ""),
+  },
```

Comment:
You don't need to create another computed field just to find a specific author. You can just use his name.